<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>708</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1037</width>
    <height>672</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Алгоритм построения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="AlgorithmTypeBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Цифрового дифференциального анализатора</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Брезенхема с действительными данными</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Брезенхема с целочисленными данными</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Брезенхема с устранением ступенчатости</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Алгоритм Ву</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Алгоритм, использующий библиотечную функцию</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="title">
         <string>Параметры отрезка</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Координаты начала отрезка</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="startCoordY">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>80</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="startCoordX">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Yн</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Xн</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Xк</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>150</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Yк</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QTextEdit" name="endCoordY">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>170</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>261</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Координаты конца отрезка</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="endCoordX">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>170</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>210</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Угол поворота (в градусах)</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="angleInput">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>271</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>180</height>
         </size>
        </property>
        <property name="title">
         <string>Цвет</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Цвет отрезка</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Цвет фона</string>
         </property>
        </widget>
        <widget class="QPushButton" name="changeSegColorBtn">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
        <widget class="QPushButton" name="changeBackgroundColorBtn">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
        <widget class="QLabel" name="segmentColor">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: black</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="backgroundColor">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: white</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plotSegmentBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Построить отрезок</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plotSpectrumBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Построить спектр</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Очистить плоскость</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
